CPS-?/README.md

```diff
@@ -0,0 +1,75 @@
+---
+CPS: ?
+Title: Domain Information Resolving Standard
```

Comment:
```suggestion
Title: Domain Name Resolution
```
This has become a problem definition rather than a `Standard``... although one or more of the emergent CIPs may eventually come to be seen as standards.

If there is other "Information" besides the domain _name_ that needs to be resolved, then I would encourage further correction from the author & reviewers. 😎